build.rs

```diff
@@ -1170,6 +1172,13 @@ fn thread_main() {
         .size_t_is_usize(true)
         .parse_callbacks(Box::new(Callbacks));
 
+    if is_wasm_emscripten {
+        let emsdk_path = PathBuf::from(env::var("EMSDK").unwrap());
```

Comment:
Is this guaranteed to be set?

At very least I'd prefer it to be `.expect()` with a message that explains what it should contain.

Reply:
And may add document about `target=wasm32-unknown-emscripten` in README.md or somewhere else? 
User need to install EMSDK and activate it, and compiling FFmpeg with Emscripten also not easy